graylog2-web-interface/src/views/pages/NewDashboardPage.jsx

```diff
@@ -31,7 +32,7 @@ const NewDashboardPage = ({ route, location, loadingViewHooks, executingViewHook
     const { state = {} } = location;
     const { view: searchView } = state;
     if (searchView && searchView.search) {
-      const query = location;
+      const { query } = location;
```

Comment:
When I go it right `processHooks` expects the query object instead of the full location.